cloud/src/autogluon/cloud/predictor/predictor.py

```diff
@@ -897,3 +951,125 @@ def _get_local_predictor_cls(self):
         from autogluon.text import TextPredictor
         predictor_cls = TextPredictor
         return predictor_cls
+
+
+class ImageCloudPredictor(CloudPredictor):
+
+    predictor_file_name = 'ImageCloudPredictor.pkl'
+
+    @property
+    def predictor_type(self):
+        return 'image'
+
+    @property
+    def _realtime_predictor_cls(self):
+        return AutoGluonImageRealtimePredictor
+
+    def _get_local_predictor_cls(self):
+        from autogluon.vision import ImagePredictor
+        predictor_cls = ImagePredictor
+        return predictor_cls
+
+    def fit(
+        self,
+        predictor_init_args,
+        predictor_fit_args,
+        image_path,
+        **kwargs
+    ):
+        super().fit(
+            predictor_init_args,
+            predictor_fit_args,
+            image_path=image_path,
+            **kwargs
+        )
+
+    def predict_real_time(self, test_data, test_data_image_column=None, accept='application/x-parquet'):
```

Comment:
Do we accept only parquet? SM API [seems support])(https://sagemaker.readthedocs.io/en/stable/api/inference/predictors.html#sagemaker.predictor.Predictor.accept) multiple accept types.

Reply:
We support more than that. This is just the default